archai/nlp/nvidia_transformer_xl/nvidia_utils/lm_iterators.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, data, bsz, bptt, device='cpu', mem_len=None, ext_len=None, wa
             self.data = torch.cat((warmup_data, self.data))
 
         # Partition data for DistributedDataParallel
-        world_size = nv_utils.distributed.get_world_size()

```

Comment:
I would suggest we keep nv_utils prefix because distributed is rather common name. In general, when we reference relatively common names, it's good for readability and avoid potential future collisions.

Reply:
Sure, sounds great! Just updated the PR.